src/domain_factory/account/brokerage_account.py

```diff
@@ -0,0 +1,40 @@
+from abc import abstractmethod
+
+from domain_factory.generatable import Generatable
+
+
+class BrokerageAccount(Generatable):
+    """ Abstract Class to generate brokerage accounts. Generate method will
+    generate the attributes shared by all brokerage account child classes.
+
+       Account [dir]
+        ->Brokerage Account [abstract]
+          -> Internal Counterparty [impl]
+          -> External Counterparty [abstract]
+            -> Contra (ECP) [impl]
+            -> ICP Holding [impl]
+            -> Depot Claim [impl]
+          -> Depot [impl]
```

Comment:
Could be wrong: impl or conc? Abstract vs Concrete is usual convention?

Reply:
done